mlflow/tensorflow.py

```diff
@@ -76,4 +94,10 @@ def log_saved_model(saved_model_dir, signature_def_key, artifact_path):
 
 
 def load_pyfunc(saved_model_dir):
+    """Load model stored in python-function format. Allows for use of the `predict(pandas DataFrame)` function.
```

Comment:
Wording suggestion: "Allows for use ..." -> "The loaded model object exposes a `predict(pandas DataFrame)` method that returns a Pandas DataFrame containing the model's inference output on an input DataFrame